packages/bootstrap/scss/scheduler/_layout.scss

```diff
@@ -25,6 +25,9 @@
         .k-event-actions {
             right: $padding-x-sm;
             font-size: $font-size-sm;
+            &:first-child {
+                margin: .1ex .4ex 0 $padding-x-sm;
```

Comment:
After much consideration and comparison to how this [has been styled in the default theme](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/scheduler/_layout.scss#L461) , the `ex` unit for the top margin is left hard-coded.